spec/requests/recipe_food_request_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'RecipeFoods', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get recipe_recipe_foods
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Great implementation team 👍🏽 However, 5 tests are failing and you need to check them.
![Screenshot from 2022-09-10 01-02-28](https://user-images.githubusercontent.com/58692590/189460549-ce7f96c3-a9b4-4b0f-ad49-7492effe4a8e.png)

Reply:
Resolved pending itens. Thank you @Whoistolu